src/pattern-library/components/patterns/ColorFoundations.js

```diff
@@ -1,74 +1,48 @@
+import tailwindConfig from '../../../../tailwind.config';
+
 import Library from '../Library';
 
 // TODO:
 // - color descriptions and guidance
-// - hex and other metadata about colors
 // - foreground color examples
-// - more examples of overriding with `!` classes
 // - valid contrast combinations
 
-const brand = ['brand'];
-
-const greys = [
-  'white',
-  'grey-0',
-  'grey-1',
-  'grey-2',
-  'grey-3',
-  'grey-4',
-  'grey-5',
-  'grey-6',
-  'grey-7',
-  'grey-8',
-  'grey-9',
-];
-
-const states = ['success', 'notice', 'error'];
+const colorConfig = tailwindConfig.theme.extend.colors ?? {};
 
-function ColorSwatch({ colorName }) {
+function ColorSwatch({ colorKey }) {
+  const colorValue =
+    typeof colorConfig[colorKey] === 'string'
+      ? `: ${colorConfig[colorKey]}`
+      : '';
   return (
     <div>
-      <div
-        className={`hyp-u-bg-color--${colorName}`}
-        style="width:16rem;height:2rem;margin-right:1em"
-      />
+      <div className={`bg-${colorKey} h-12 mr-4`} />
```

Comment:
In case it's not clear, it's tailwind magically providing `bg-<color>` classes here. It also gives you `text-<color>`, `border-<color>`, etc.